pkg/scheduler/controller/shoot/scheduler_control.go

```diff
@@ -151,12 +151,11 @@ func (c *defaultControl) ScheduleShoot(ctx context.Context, obj *gardencorev1bet
 		if _, ok := err.(*common.AlreadyScheduledError); ok {
 			return nil
 		}
-		c.reportFailedScheduling(shoot, err)
```

Comment:
I'm concerned that we remove this user feedback. Errors can occur during the seed assignment, for example if [DNS is misconfigured](https://github.com/gardener/gardener/blob/056f925a09162fb90fcce938b04c0397250bf25f/plugin/pkg/shoot/dns/admission.go#L207) and with this change the user won't notice such issues.

Reply:
Hm, good point. The problem is that these events easily pile up due to the retry of the scheduler. This may lead to an extensive volume of events. Probably it makes sense to also delete the events of type `Warning` in the event controller if they are older than a certain time?
Or, if there are "too many"' (whatever that is) other `Warning` events with a newer timestamps already, then the event controller could clean up the older ones.
/cc @timebertt

Reply:
Well, I think these events would then not be stored separately in etcd, but rather the eventCount would be increased and the lastTimestamp refreshed, right?

Reply:
Is it? Then it's all fine :)

Reply:
I would think so: https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/383-new-event-api-ga-graduation
@BeckerMax can you confirm?

Reply:
Yes, in this case the events are grouped together to a single object. From https://github.com/kubernetes/community/blob/master/contributors/design-proposals/instrumentation/events-redesign.md :
```
Deduplication logic groups together Event which have the same:

- Source Component and Host
- InvolvedObject Kind, Namespace, Name, API version and UID,
- Type (works as Severity)
- Reason
```